### PR TITLE
Added ZIO SQL

### PR DESCRIPTION
**Pull request by sviezypan**
Hello @adamw 
I am currently writing a blog post about ZIO SQL and I used example data from your old blog post (cities, metrolines...)

I was hoping we could merge this PR which adds ZIO SQL to the mix, so people that are interested could compare all the libraries under one repository.  In my blog, I included link to this repo and your blog.

### CODE REVIEW

File: ziosql/src/main/resources/init.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
+DROP TABLE IF EXISTS metro_line;
+DROP TABLE IF EXISTS metro_system;
+DROP TABLE IF EXISTS city;
+
+CREATE TABLE city(
```
Comment (adamw) [inline ziosql/src/main/resources/init.sql]: can't you use the same schema/data as the others? (there is a dependency on `common` so I suppose this should work ...)

File: ziosql/src/main/scala/com/softwaremill/sql/ZioSqlTests.scala [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+    def deleteCity(id: CityId) = 
+        deleteFrom(city)
+            .where(cityId === id.id)
+
+    // TRANSACTIONS
+
+    val i = CityId(4)
+
+    val transaction = for {
```
Comment (adamw) [inline ziosql/src/main/scala/com/softwaremill/sql/ZioSqlTests.scala]: this example looks different from the other "transaction" ones. Are you running a single transaction here, or are there a couple of them?

**Comment by adamw** [comment]
Thanks :) probably the others would need updating too. I haven't read each example in depth, are they the same examples as in the other implementations?

File: ziosql/src/main/resources/init.sql (hunk 1 above)
Comment (sviezypan) [reply ziosql/src/main/resources/init.sql]: done

File: ziosql/src/main/scala/com/softwaremill/sql/ZioSqlTests.scala (hunk 2 above)
Comment (sviezypan) [reply ziosql/src/main/scala/com/softwaremill/sql/ZioSqlTests.scala]: its one transaction, I modified the example to look like the other ones

**Comment by sviezypan** [comment]
Yes I compared with Doobie and I implemented all the examples except for executing query as raw String -  ZIO SQL cannot do that at the moment. It would be possible with ZIO JDBC, however these libraries are not integrated yet.

**Comment by adamw** [comment]
Thanks :) Looks good, though I didn't read each example that carefully. Maybe one day when I'll revise my own blog. Let me know when yours comes out!

**Comment by sviezypan** [comment]
Thanks @adamw, blog post is already out https://scalac.io/blog/zio-sql-type-safe-sql-for-zio-applications/

**Pull request merged**